rpcs3/Emu/RSX/Common/texture_cache.h

```diff
@@ -98,6 +87,11 @@ namespace rsx
 			image_type = type;
 		}
 
+		void set_sampler_status(const rsx::texture_sampler_status status)
```

Comment:
missing ref?

Reply:
its an enum value so no. Its ever used with allocated vars

Reply:
I see, `const` in signature confused me since it's not needed here (see my other comment) and usually considered bad style.